src/containers/search/SearchForm.tsx

```diff
@@ -80,7 +75,7 @@ export const SearchForm: React.FC<{ closed?: boolean }> = ({ closed }) => {
   const { pathname, search } = useLocation();
 
   const [open, setOpen] = useState<boolean>(!closed);
-  const isMobile = useMediaQuery(theme.breakpoints.down('md'));
+  const isMobileView = useMediaQuery(theme.breakpoints.down('md'));
```

Comment:
`isMobileView` is technically not the correct name, as this is also includes tablet.

I would go for `isDesktop` and make it falsey if you want to do the check throughout the code

Reply:
`down("sm") alternatively you want to keep it calling `isMobile`

Reply:
@silksil This has been used in many more places than just here. This can be refactored in a subsequent PR to address all appearances